app/views/appointments/new.html.slim

```diff
@@ -25,8 +26,7 @@
   br/
   = f.submit'Create', :class => 'btn btn-success', :id =>'create'
   = link_to 'Cancel',admin_index_path, :class => 'btn btn-default'
-
 javascript:
```

Comment:
avoid javascripts from html templates. js code specific to appointments/new should be in the respective .js file. For generic cases, such as the selectpicker below have it in a file, and include in application.js